### PR TITLE
Add AddLocalCampaign code example

### PR DESCRIPTION
**Pull request by PierrickVoulet**
Change-Id: Ibedf75cae6bc0d1716ca02a53a2884cf70cb7d67

### CODE REVIEW

File: examples/AdvancedOperations/AddLocalCampaign.php [hunk 1]
```diff
@@ -0,0 +1,459 @@
...
+     * @return string the campaign budget resource name
+     */
+    private static function createCampaignBudget(
+        GoogleAdsClient $googleAdsClient,
+        int $customerId
+    ) {
+        $campaignBudget = new CampaignBudget([
+            'name' => 'Interplanetary Cruise Budget #' . uniqid(),
+            'delivery_method' => BudgetDeliveryMethod::STANDARD,
+            'amount_micros' => 500000,
```
Comment (fiboknacky) [inline examples/AdvancedOperations/AddLocalCampaign.php:169]: Is it intended to use the different number than that of the golden example (50000000)?

File: examples/AdvancedOperations/AddLocalCampaign.php [hunk 2]
```diff
@@ -0,0 +1,459 @@
...
+            // A Local campaign cannot use a shared campaign budget.
+            'explicitly_shared' => false
+        ]);
+
+        // Creates a campaign budget operation.
+        $campaignBudgetOperation = new CampaignBudgetOperation();
+        $campaignBudgetOperation->setCreate($campaignBudget);
+
+        // Issues a mutate request to add campaign budgets.
```
Comment (fiboknacky) [inline examples/AdvancedOperations/AddLocalCampaign.php:178]: Nit: "add the campaign budget" (no s)

File: examples/AdvancedOperations/AddLocalCampaign.php [hunk 3]
```diff
@@ -0,0 +1,459 @@
...
+                    OptimizationGoalType::DRIVING_DIRECTIONS
+                ]
+            ])
+        ]);
+
+        // Creates a campaign operation.
+        $campaignOperation = new CampaignOperation();
+        $campaignOperation->setCreate($campaign);
+
+        // Issues a mutate request to add campaigns.
```
Comment (fiboknacky) [inline examples/AdvancedOperations/AddLocalCampaign.php:249]: Ditto for campaign"s"

File: examples/AdvancedOperations/AddLocalCampaign.php [hunk 4]
```diff
@@ -0,0 +1,459 @@
...
+            'name' => 'Earth to Mars Cruises #' . uniqid(),
+            'campaign' => $campaignResourceName,
+            'status' => AdGroupStatus::ENABLED
+        ]);
+
+        // Creates an ad group operation.
+        $adGroupOperation = new AdGroupOperation();
+        $adGroupOperation->setCreate($adGroup);
+
+        // Issues a mutate request to add the ad groups.
```
Comment (fiboknacky) [inline examples/AdvancedOperations/AddLocalCampaign.php:295]: Ditto: ad group"s"

File: examples/AdvancedOperations/AddLocalCampaign.php [hunk 5]
```diff
@@ -0,0 +1,459 @@
...
+                    )])]
+                ])
+            ])
+        ]);
+
+        // Creates an ad group ad operation.
+        $adGroupAdOperation = new AdGroupAdOperation();
+        $adGroupAdOperation->setCreate($adGroupAd);
+
+        // Issues a mutate request to add the ad group ads.
```
Comment (fiboknacky) [inline examples/AdvancedOperations/AddLocalCampaign.php:361]: Nit: ad"s".

**Review by fiboknacky — CHANGES_REQUESTED** [review]

File: examples/AdvancedOperations/AddLocalCampaign.php (hunk 1 above)
Comment (PierrickVoulet) [reply examples/AdvancedOperations/AddLocalCampaign.php]: It was not intended, great catch!

**Review by fiboknacky — APPROVED** [review]

**Review by wfansh — APPROVED** [review]

**Pull request merged**